.github/workflows/test.yml

```diff
@@ -67,43 +31,21 @@ jobs:
         run: |
           isort .
 
-  test-postgres:
```

Comment:
The changes here seem to incorrect.

The `DATABASE_URL` isn't passed so the app will run with sqlite, although the tox test appear to be running with postgres I don't think thats the case.

@katdom13 I don't know the answer immediately but you can leave it with me to look at.